src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -70,13 +67,13 @@ public Person getPersonById(int id)  {
 		try {
 			 
 			p = dao.getOne(id);
-			System.out.println("This is my P: "+p);
+			IntEvalLogger.LOGGER.info("This is my P: "+p);
 			 
-		 } catch (Exception e) {		
+		 } catch (EntityNotFoundException e) {		
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1166)